FWCore/ParameterSet/python/Config.py

```diff
@@ -533,11 +533,17 @@ def extend(self,other,items=()):
                 if not item.hasLabel_() :
                     item.setLabel(name)
             elif isinstance(item,Schedule):
-                self.__setattr__(name,item)
```

Comment:
I'm not sure I'm happy with this idea. Everything else in the configuration gets a replacement if you set its name except for Schedule?